src/components/Search/SubredditForm.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import * as S from './SubredditForm.style';
+/* eslint-disable react/prop-types */
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
Why are these lint rules disabled? Usually it would be better to try to fix them

Reply:
Yea I figured it out, i removed those lines 👍